common/test/acceptance/tests/video/test_video_module.py

```diff
@@ -554,6 +589,46 @@ def test_video_language_menu_working(self):
         self.video.select_language('en')
         self._verify_caption_text('Welcome to edX.')
 
+    def test_video_language_menu_working_closed_captions(self):
+        """
+        Scenario: Language menu works correctly in Video component, checks closed captions
+        Given the course has a Video component in "Youtube" mode
+        And I have defined multiple language transcripts for the videos
+        And I make sure captions are closed
+        And I see video menu "language" with correct items
+        And I select language with code "zh"
+        Then I see "好 各位同学" text in the closed captions
+        And I select language with code "en"
+        Then I see "Welcome to edX." text in the closed captions
+        """
+        self.assets.extend(['chinese_transcripts.srt', 'subs_3_yD_cEKoCk.srt.sjson'])
+        data = {'transcripts': {"zh": "chinese_transcripts.srt"}, 'sub': '3_yD_cEKoCk'}
+        self.metadata = self.metadata_for_mode('youtube', additional_data=data)
+
+        # go to video
+        self.navigate_to_video()
+        self.video.show_closed_captions()
+
+        correct_languages = {'en': 'English', 'zh': 'Chinese'}
+        self.assertEqual(self.video.caption_languages, correct_languages)
+
+        # Change the language to Chinese and check for correct text
+        self.video.select_language('zh')
+        self.video.click_player_button('play')
+        self.video.wait_for_position('0:05')
```

Comment:
Not sure, but don't think so. The video plays, it just plays choppy (at least locally), so sometimes 00:01 doesn't have the text, but other times it might. I chose a time that should guarantee text.